unime/src/routes/+layout.svelte

```diff
@@ -39,23 +39,24 @@
 
   const systemColorScheme = window.matchMedia('(prefers-color-scheme: dark)');
 
-  systemColorScheme.addEventListener(
-    'change',
-    (e) => {
-      if ($state?.profile_settings.profile?.theme) {
-        determineTheme(e.matches, $state.profile_settings.profile.theme);
-      }
-    },
-    { once: true },
-  );
+  // This is used for detecting the OS theme changes.
```

Comment:
I don't think this comment adds any value since the variable is already called `systemColorScheme`.